### PR TITLE
dev-util/cargo-license: Version bump to 0.5.1

### PR DESCRIPTION
**Pull request by etc404**
Signed-off-by: Dylan Wolf <etc404@protonmail.com>

### CODE REVIEW

**Comment by gyakovlev** [comment]
Ebuild looks good from first glance, however there are some issues with PR, everything easily fixable, you are almost there.

* you need to add signed-off-by to your commit (`git commit --signoff`), and not just add it to PR test.
if you need to fix existing commit - run `git commit --amend --signoff --no-edit`
* you need to properly format commit message: your PR title is correct but commit message itself is not. again `git commit --amend` and re-edit. after edit is done you have to `git push --force`
* you opened PR from master branch - never do that. before you start working on something - branch it.
`git checkout -b `cargo-audit-bump` (branch name can be anything, every PR is a new branch, you can delete it after it's merged)
...do work
`git push --set-upstream origin cargo-audit-bump` 
and after that you open a PR from your `cargo-audit-bump` branch.
* we do not allow merge commits, ever. 
run this in your gentoo.git checkout:
```
git config --local user.name "Your Full Name"
git config --local user.email "example@gentoo.org"
git config --local pull.rebase true
git config --local --unset pull.ff
git config --local push.default simple
```
and check the rest of git setup tips here:
https://wiki.gentoo.org/wiki/Gentoo_git_workflow

TLDR: I recommend closing this PR, applying our git settings, moving your work to a branch and opening a new PR from a branch with things I've mentioned fixed. that's ok for a first PR,
